### ISSUE TITLE
Sort image in column header is hard to see 

### ISSUE THREAD

**Issue by donnrri** [rstudio/DT#839]
The sorting icon in the data-table column header is quite lightly colored, for me with poor eyesight it is hard to see. Further it cannot be re-styled to improve its accessibility.

Could a different image be used or even better an SVG.  I will volunteer to do the required work if you like as I think this is an excellent project.

Thanks for your time.

**Comment by stla** [comment]
Here is a way to set another icon:

```
library(shiny)
library(DT)

css <- HTML(
  "table.dataTable thead .sorting {",
  "  background-image: url(https://image.flaticon.com/icons/svg/25/25756.svg) !important;",
  "  background-size: 15px;",
  "}"
)

ui <- fluidPage(
  tags$head(tags$style(css)),
  br(),
  DTOutput("dtable")
)

server <- function(input, output){
  output[["dtable"]] <- renderDT({
    datatable(iris)
  })
}

shinyApp(ui, server)
```

**Comment by shrektan** [comment]
@donnrri Thanks for the reporting. 

According to my knowledge, for now, it's not possible to change the sorting icon natively (via the options). I mean, you can only change it as @stla suggested, by hacking the CSS.

DT is only an R-wrapper of the Javascript library [DataTables](https://github.com/DataTables). Changing the theme directly in DT seems not a very good idea to us, as it introduces inconsistency with the upstream while it could be easily fixed by providing a CSS style sheet, as @stla mentions. 

The good news is, we may be able to do the customized theming in the future (see PR #740). Then we may be able to do this via an option. 

So I'm going to close this issue for now but feel free to reopen it if you have other ideas.

Thanks.
